### PR TITLE
V1/75 answer poll

### LINKED ISSUE #75: Answer poll
Creat laravel controller for poll answering
As a user
I Want to answer a poll
So I can give my opinion

**Estimation: 1h**

### PR DESCRIPTION
**Pull request by kayoumido**
Normally, I've implemented the following validations:
- Only attendees can vote
- A user can't vote twice for the same Poll
- The choice must be related to the poll
- The poll has to be "votable" i.e it's status is set to `published`

### CODE REVIEW

File: src/app/Http/Controllers/API/PollController.php [hunk 1]
```diff
@@ -171,12 +175,19 @@ public function results(Request $request, Poll $poll)
...
+     * @param VoteRequest $request
      * @param Poll $poll
      * @param User $user
      */
-    public function vote(Request $request, Poll $poll, User $user)
+    public function vote(VoteRequest $vote_request, Poll $poll, User $user)
     {
-        //
+        $user->polls()->attach($poll, [
+            'choice_id' => $request->choice_id,
```
Comment (Serphentas) [inline src/app/Http/Controllers/API/PollController.php:185]: `$request` is undefined, only works when changed to `$vote_request`

**Review by Serphentas — CHANGES_REQUESTED** [review]
Tested:
- [x] Voting on poll not published, denied
- [x] Voting on poll not attendee, denied
- [x] Voting on poll with other poll's choice, denied
- [x] Voting on poll twice, denied
- [x] Voting on poll with other user id, denied
- [x] Voting on poll once with related poll, authorized

File: src/app/Http/Controllers/API/PollController.php (hunk 1 above)
Comment (kayoumido) [reply src/app/Http/Controllers/API/PollController.php:185]: mb

Comment (kayoumido) [reply src/app/Http/Controllers/API/PollController.php:185]: fixed in a77e3cac6f5a711970fea6f4f8ed25d0dcaa8d1d

Comment (Serphentas) [reply src/app/Http/Controllers/API/PollController.php:185]: All good

**Review by fabio10000 — APPROVED** [review]

**Review by Serphentas — APPROVED** [review]

**Pull request merged**